app-text/texlive-core/texlive-core-2023-r9.ebuild

```diff
@@ -433,7 +433,10 @@ src_install() {
 	use doc || rm -rf "${ED}/usr/share/texmf-dist/doc"
 
 	dodir /etc/env.d
-	echo 'CONFIG_PROTECT_MASK="/etc/texmf/web2c /etc/texmf/language.dat.d /etc/texmf/language.def.d /etc/texmf/updmap.d"' > "${ED}/etc/env.d/98texlive"
+	echo 'CONFIG_PROTECT_MASK="/etc/texmf/web2c' \
```

Comment:
Does that do the right thing?

In any case, I would prefer using newenvd and a bash here doc:

```bash
newenvd - 98texlive <<-EOF
CONFIG_PROTECT_MASK="/etc/texmf/web2c /etc/texmf/language.dat.d /etc/texmf/language.def.d /etc/texmf/updmap.d"
EOF
```